instrumentation/github.com/aws/aws-sdk-go-v2/otelaws/example/go.mod

```diff
@@ -0,0 +1,21 @@
+module go.opentelemetry.io/contrib/instrumentation/github.com/aws/aws-sdk-go-v2/otelaws/example
+
+go 1.14
+
+replace (
+	go.opentelemetry.io/contrib => ../../../../../..
+	go.opentelemetry.io/contrib/instrumentation/github.com/aws/aws-sdk-go-v2/otelaws v0.18.0 => ../
+	go.opentelemetry.io/contrib/propagators/aws v0.18.0 => ../../../../../../propagators/aws
```

Comment:
This isn't a required package, is a replace needed?

Reply:
yes, I use it in my e2e test and forget to remove it.